kibana/connector.py

```diff
@@ -72,6 +73,16 @@ def version(self):
         if self.status:
             return self.status.get("version", {}).get("number")
 
+    @staticmethod
+    def ndjson_file_data_prep(lines: List[dict], filename: str) -> (dict, str):
```

Comment:
I really hated having to do this, but there may be a conflict with how `requests` generates file contents vs what Kibana is expecting. I tried many ways, with only manual explicit setup like this working.

Open to collapsing down to pure request if possible

Reply:
What issue did you run into?

Reply:
constantly 500 with `hapi unable to read stream`. `hapi` is the library used in Kibana to process the stream.

Reply:
I think the issue is that you supplied raw_data, and would be using the `file` parameter as seen in the [docs](https://www.elastic.co/guide/en/security/current/rules-api-import.html#_request_url_14).

I think you'll have to update the kibana.connector.Kibana.request method to support `file` and pass down.